app/config.js

```diff
@@ -1,13 +1,13 @@
 require.config({
```

Comment:
Is this file needed at all anymore? I thought with the new `import`s we didn't need this